### ISSUE TITLE
Probleme de fonctionnement des commmandes pour android

### ISSUE THREAD

**Issue by magicbret** [ldleman/yana-server#154]
Hello,

Je relance le sujet posté par HARRYJC au sujet de problème avec l'appli android (https://github.com/ldleman/yana-server/issues/152). 
Au démarrage de l'appli : "Echec de compréhension par rapport à la réponse du Raspberry Pi. Veuillez présenter le problème à Nover"
- Rien dans les logs lighttpd
- Désinstallation du plugin météo (au cas où, mais çà n'a rien changé)
- Reboot, réinstallation de yana

Tout fonctionnait correctement avant la réinstallation du serveur Yana suite à un crash de la carte SD du Raspberry :-(

**Comment by jvanpou86** [comment]
Bonjour,
Je rencontre le meme problème avec le update des commandes sur android.

**Comment by ldleman** [comment]
Avez vous essayé de contacter nover pour voir d'ou peux venir le problème?

**Comment by rikimaruneo** [comment]
Bonjour je vient de voir sur yana android que nover avez mis un message comme quoi l'aplication etait obsolete avec les derniere mise a jour de yana serveur 
une question comment avoir les encienne version 

merci

**Comment by ldleman** [comment]
Si tu fais ça tu n'aura plus les débug, mise à jours etc... je te conseille plutot de télécharger la bêta du client android v2.0 sur https://play.google.com/store/apps/details?id=alexandreroland.yana

Si tu teins quand même a revenir à l'ancienne version, tu peux revenir aux commit avant modif pour la v2.0 avec les commandes

```git fetch origin e1b65bb8b441a992e093283e818396e009511bfc
git reset --hard FETCH_HEAD```

**Comment by jvanpou86** [comment]
Merci Idleman!
Effectivement, avec la nouvelle version beta sur Androïd ça fonctionne très bien.

Sur le Raspberry ,
-J'ai simplement lancé 'GIT PULL' pour vérifier et ça m'a confirmé que j'étais à jour.
  (Pourtant je ne trouve pas l'indication de version Yana4All 0.1 Beta ?
   Dans la configuration il est indiqué PROGRAM_VERSION = 3.0.6 ?)
-J'ai lancé le PHP socket.php (sans rien modifer) 
Sur le Galaxy,
-Installation de Yana v2 par Alexandre Roland (Merci à lui en passant et Bravo)
-Saisie IP serveur, Token et Nom.
-Connexion.... 

Et voilà, les commandes vocales fonctionnent et je peux allumer les lampes, ouvrir et fermer les volets, etc...
Ravi !

**Comment by ldleman** [comment]
Fais pas attention aux numéros de version, c'est un changement manuel , et comme je l'oublie constamment, ça ne veux plus rien dire ^^.

Niquel pour yana android, n'hésite pas a faire tes retours et compliments a Alexandre qu'il puisse améliorer sa chouette petite appli :).

De mémoire Alexandre m'a dit que ça fonctionnais aussi pour le WAN, suffit de changer l'ip et de débloquer le port 9999 sur ton routeur.

**Comment by jvanpou86** [comment]
Bonjour,
Comme je le disais plus haut, l'application YANA4ALL d'Alexandre sur Androïd fonctionne parfaitement sur mon Galaxy S4, par contre sur un Galaxy Core 4G non (incompatible avec cette version).
Depuis, ayant fais parvenir mes remarques à Alexandre, il a eu la gentillesse d'ajouter la prise en compte LAN ou WAN selon la détection ou non du SSID Wifi.
Bravo à Alexandre Roland pour cette excellente appli.

Maintenant, j'ai 'bricolé' avec le plugin Evènements pour avoir des evènements dont les horaires se mettent à jour automatiquement avec les horaires solaires. C'est pratique pour des volets par exemple sans ajouter quoi que ce soit matériellement.
Il suffit d'un script PHP lancé par une ligne CRON qui utilise les fonctions Sun-Set() et Sun-Rise() et qui modifie les enregistrement de la base SQLite.

**Comment by juliendiotworldtour** [comment]
Petite question de mon côté pour @Charal ! 
Il semblerait que lappli Yana4all ne puisse pas prendre en compte le plugin switchi. J'ai contacté Alexandre et le problème viendrait du plugin.
Aussi, mes commandes vocales SWITCHI ne marchent pas depuis Yana server tant que je n'ai pas activé séparément chaque GPIO au moins une fois avant.

Bonne.journée ! Et un grand merci à tous !

**Comment by juliendiotworldtour** [comment]
Bonjour @ldleman , il semblerait que @Charal soit pas mal pris depuis 10 jours car il me laisse sans réponse. Le problème que je rencontre est le suivant: l'appli Android YANA4All ne peut pas prendre en compte les commandes vocales groupées de SWITCHI (plusieurs activation de GPIO avec une seule phrase). As tu une idée du pourquoi du comment?

Je te remercie d'avance!

**Comment by Charal** [comment]
Salut Julien, 

Désolé j'ai beaucoup de boulot en ce moment. 
J'essaye de m'y pencher ce week end, je pense pas qu'il y ai de raison que cela vienne de l’application Android. 
Tu parles donc des commandes vocales généré pour chacun des groupes que tu crées. 
Je regarderais, mais bon, au pire utilise l'ancien client vocal, il était lent mais avait le mérite de fonctionner. 
Bonne journée. 

@lex 


----- Mail original -----

De: "juliendiotworldtour" <notifications@github.com> 
À: "ldleman/yana-server" <yana-server@noreply.github.com> 
Cc: "Charal" <alex.rault@laposte.net> 
Envoyé: Lundi 29 Février 2016 13:08:18 
Objet: Re: [yana-server] Probleme de fonctionnement des commmandes pour android (#154) 



Bonjour @ldleman , il semblerait que @Charal soit pas mal pris depuis 10 jours car il me laisse sans réponse. Le problème que je rencontre est le suivant: l'appli Android YANA4All ne peut pas prendre en compte les commandes vocales groupées de SWITCHI (plusieurs activation de GPIO avec une seule phrase). As tu une idée du pourquoi du comment? 

Je te remercie d'avance!

**Comment by juliendiotworldtour** [comment]
Salut @alex, merci pour ton retour.

Non le problème ne vient pas de yana4all mais de switchi car les autres plugins (story et commandes vocales) fonctionnent.
Je n'arrive pas à faire fonctionner les commandes vocales pour les associations groupées comme "YANA, allume/éteint tout"

Merci !

**Comment by ldleman** [comment]
Yop, le problème est il réglé ? Je peux clore ? @Charal @juliendiotworldtour

**Comment by Charal** [comment]
Non, je suis en train de traiter avec Alex, et non je ne suis pas Schizophréne :) 
@lex 

----- Mail original -----

De: "Idleman" <notifications@github.com> 
À: "ldleman/yana-server" <yana-server@noreply.github.com> 
Cc: "Charal" <alex.rault@laposte.net>, "Mention" <mention@noreply.github.com> 
Envoyé: Vendredi 20 Mai 2016 13:16:36 
Objet: Re: [ldleman/yana-server] Probleme de fonctionnement des commmandes pour android (#154) 



Yop, le problème est il réglé ? Je peux clore ? @Charal @juliendiotworldtour
